mldsa/poly.c

```diff
@@ -346,6 +347,53 @@ void poly_uniform(poly *a, const uint8_t seed[MLDSA_SEEDBYTES], uint16_t nonce)
   }
 }
 
+void poly_rej_uniform_4x(poly *vec,
```

Comment:
I assume you have copied that from mlkem-native? Can you edit the license header?

Reply:
Fixed